### ISSUE TITLE
Use large preview by default in social cards

### ISSUE THREAD

**Issue by kof** [webstudio-is/webstudio#3104]
By default twitter renders a card like this 
<img width="573" alt="image" src="https://github.com/webstudio-is/webstudio/assets/52824/fb193029-e579-4e93-b4f8-2828d074362e">

But we want it like this: 
<img width="439" alt="image" src="https://github.com/webstudio-is/webstudio/assets/52824/f37d1f81-918a-428a-9089-6741718c1b04">

Meta needed to be set by default:

twitter:card content: summary_large_image

**Comment by JayaKrishnaNamburu** [comment]
<img width="444" alt="Image" src="https://github.com/user-attachments/assets/8b14e1f0-69da-4b91-a34c-4daf1810ad39" />

@kof currently, the builder doesn't add any `og` tags unless explicitly specified right. So, should we check if a user is explicitly adding `twitter:card` and respect it. If not, just add the `twitter:card` and `content:summary_large_image`. So, by default we can force twitter links to behave with bigger images when shared ?
https://developer.x.com/en/docs/x-for-websites/cards/overview/summary-card-with-large-image

**Comment by JayaKrishnaNamburu** [comment]
> Makes sense to not change something user has explicitely defined. I was sure we are adding og tags by default though

Yup, the builder is adding og tags. But not the twitter ones. Unless explicitly specified, they are left to the platforms and no defaults in the builds.

**Comment by JayaKrishnaNamburu** [comment]
Yeah, this is going to be a bloat. By default the sdk is setting only the `og` ones and letting platforms interpret them. And allows users to add anything extra.
https://github.com/webstudio-is/webstudio/blob/be6b5fa9660543dfeb5c688cd269433f02498e18/packages/react-sdk/src/page-settings-meta.tsx#L63

In summar, if we need to make this happen. We need to add 
```xm
<meta name="twitter:card" content="summary_large_image">
```

Which is going to be bloat unless the users are adding anything specific to twitter. KInd of adding a default behaviour on the builder side.

**Comment by kof** [comment]
yes yes, do it, I just meant to not add all twitter tags of those are correctly interpreted
